source/server-selection/server-selection.rst

```diff
@@ -1211,7 +1229,17 @@ Pseudocode for `multi-threaded or asynchronous server selection`_::
 
             if servers is not empty:
                 in_window = servers within the latency window
-                selected = random entry from in_window
+                server1, server2 = random two entries from in_window
+
+                diff = abs(server1.pool.active_connection_count - server2.pool.active_connection_count)
+                if diff > max(0.05 * max_pool_size, 1):
+                    selected = server in (server1, server2) with minimum active_connection_count
+
+                if server1.pool.available_connection_count >= server2.pool.available_connection_count:
```

Comment:
I think this `if` is supposed to be in an `elif`:
```
if diff > max(0.05 * max_pool_size, 1):
    selected = server in (server1, server2) with minimum active_connection_count
elif server1.pool.available_connection_count >= server2.pool.available_connection_count:
    selected = server1
else:
    selected = server2
```

Reply:
yep, fixed